blockchain/indexers/slpindex_test.go

```diff
@@ -0,0 +1,138 @@
+package indexers
+
+import (
+	"bytes"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"testing"
+
+	"github.com/gcash/bchd/chaincfg/chainhash"
+	"github.com/gcash/bchd/wire"
+	"github.com/simpleledgerinc/goslp"
+	"github.com/simpleledgerinc/goslp/v1parser"
+)
+
+// TestSlpMesesageUnitTests downloads SLP parser unit tests and checks the parser throws for each test where code != nil
+func TestSlpMessageUnitTests(t *testing.T) {
+	resp, err := http.Get("https://raw.githubusercontent.com/simpleledger/slp-unit-test-data/master/script_tests.json")
```

Comment:
Let's either hardcode these objects in the test or create a json file that is loaded for the test. 

We don't want this test to rely on the availability of that endpoint. Or even an internet connection for that matter.

Reply:
done.